extensions/account-registration.md

```diff
@@ -0,0 +1,205 @@
+---
+title: `account-registration` Extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Ed Kellett"
+    period: "2020"
+    email: "e@kellett.im"
+  -
+    name: "Val Lorentz"
+    period: "2021"
+    email: "progval+ircv3@progval.net"
+---
+
+
+## Motivation
+
+The lack of a standardized way to register an account has been a source
+of frustration for many participants for several years.
+
+This isn't the first attempt to fix it. Unlike the withdrawn `ACC` proposal,
+this effort aims to be more appealing to implementors by being severely
+restricted in scope.
+The only register-verify flows supported are those already in use in the wild.
+The authors hope that this stripped-down solution will serve as the basis
+for future organic expansion.
+
+### Capability
+
+This specification adds the `draft/account-registration` capability, whose
+presence signifies that the server accepts the `REGISTER` command.
+
+The capability has an optional value, a comma-separated list of key-value
+pairs; the format is intended to follow the precedent set by
+[the multiline draft][multiline].
```

Comment:
Not sure we need to mention this precedent here. The format should already be described in a standalone way.

Reply:
I don't understand; do you want me to copy-paste it here?

Reply:
I think @jwheare wants to drop this part "the format is intended to follow the precedent set by multiline", while keeping the rest of the sentence.

Reply:
"The capability has an optional value, a comma-separated list of key-value pairs" alone is not enough to describe the format.

Reply:
If you need to copy paste it here to make it standalone then yeah.